.github/workflows/bump_packages.yml

```
@@ -55,6 +55,22 @@ jobs:
         title: "Update ${{ matrix.package_name }} to ${{ matrix.new_version }}"
         body: ''
         delete-branch: true
+    - name: Send email to Discourse on failure
+      if: failure()
+      uses: dawidd6/action-send-mail@v3
+      with:
+        server_address:
+        server_port:
+        username: ${{secrets.MAIL_USERNAME}}
+        password: ${{secrets.MAIL_PASSWORD}}
```

```suggestion
        server_address: community.theforeman.org
        server_port: 25
```

no need no auth if we talk to our machine directly ;)